### DISCUSSION TITLE
Latest versions : Firmware 1.4 | Remote Script 1.2.2

### DISCUSSION THREAD

**Discussion by KBLiveSolutions** [KBLiveSolutions/open.control#6] [Announcements]
# **Firmware version 1.4 (updated 23 February 22)**
[Download and install instructions here](https://github.com/KBLiveSolutions/open.control-firmware)

Changelog:
- 1.4
   - fixed Options (Session Box, Metronome Blink) resetting when action was changed
   - Custom Color "Off" now works
- 1.3
   - fixed defaut sliders values
   - fixed double press for Looper buttons
   - fixed clear LEDs
- 1.2
   - **"Linked page"** : In the Editor, you can now "link" a Page to a View in Live. This is done with the "Link to" buttons near the Page Select chooser. More details in the [User Manual](https://kblivesolutions.github.io/open.control/user_manual/).
- 1.1
   - Changing parameters now shows a value on the Display
   - **Page #** is now displayed when switching pages

___
# **Remote Script version 1.2.1 (updated 3 February 22)**
[Download and install instructions here](https://github.com/KBLiveSolutions/open.control-remote_script)

Changelog:
- 1.2.2:
   - added colors to Markers for Setlist SONGS : just add (COLOR#) to a Marker's name
   - fixed (SONG) detection for arrangement markers
   - fixed Arrangement Marker "Launch No Q"
   
- 1.2.1:
   - Fixed Scene name not showing
   - fixed Stop All Clips
   - added Groove Amount feedback
   - added Cue Volume feedback
   - improved displayed parameter value accuracy
   - volumes are rounded to 1 decimal
   - cleaned and documented the code

- 1.2
   - **Setlists**: you can know create setlist by adding (SONG#) to a Scene or Marker name. Refer to the [User Manual](https://kblivesolutions.github.io/open.control/user_manual/) for further details.
   - Fixed Linked Page bugs

- 1.1
   - Linked Page integration
   - if a Marker has an empty Name, the Display will no longer be empty. Instead, it will show "No Name".
   - Added Fixed Length recording

**Comment by KBLiveSolutions** [comment]
Bonjour

Je n’ai rien remarqué d’anormal, peux-tu me décrire un plus précisément ce
que tu constates stp ?

**Comment by 77166** [comment]
Re salut,

Finalement j'ai réussi à me dépanner dans la nuit  et ce matin il me semblait avoir enlevé le message sur GitHub…
Ça faisait un petit moment que j'avais pas ouvert le template avec les loopers … 
le midi mapping avait bougé mais ça c'est sûrement de ma faute, j'ai dû sauvegarder par erreur après des fausses manip...  Notamment, j'avais le CC49 qui s’était glissé dans le mapping , erreur dont tu mets en garde dans ta vidéo…
Mais tout de même quelque chose m’a semblé bizarre , avec l'external midi dans Open control  Editor…
Avec Open Control câblé la partie external midi était vide .
Cet espace n'est-il pas censé refléter l'État de Open Control ...comme ci dessous 
￼
J’ai retapé toutes les valeurs car c’était vide …j’ai refait tout le midi mapping 
Mais J’avais tjs un pb avec le Button 1 que je commande avec la note C#-2 canal 4 du FCB1010. Impossible de commander les big Button des Loopers
Curieusement il me semble qu'à un moment j’ai basculé sur ce réglage ci dessous et ça a fonctionné ...
￼
Que signifie num ?

Question pour faire le mapping, je suis obligé de passer par un autre contrôleur que je mets alternativement sur les canal 16, 15, 14, et 13…etc pour les touches C0 C#0 D0 etc …je n’arrive pas à le faire directement depuis open control même quand il est cablé avec des footswitches…. 

A plus
